packages/components/src/custom-select-control/index.js

```diff
@@ -97,20 +129,36 @@ export default function CustomSelectControl( {
 	const menuProps = getMenuProps( {
 		className: 'components-custom-select-control__menu',
 		'aria-hidden': ! isOpen,
+		style: {
+			minWidth: widthMinusBorder,
```

Comment:
Why are we setting the min width, instead of the width itself?

Reply:
Similar to one of [the other comments](https://github.com/WordPress/gutenberg/pull/37272#discussion_r772072820), I thought that `minWidth` would make sense when I was calculating width from the Button / toggle width, so that the menu could expand further than the button. Now that we're calculating from the parent container, `width` makes more sense, so I've updated to `width` instead 👍